From 31a85042c6a7b558c5c6af32718a241e6f7e13eb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:41:11 +0800
Subject: [PATCH] Revert "rfkill: Introduce CONFIG_RFKILL_PM and use instead of
 CONFIG_PM to power down"

This reverts commit 3431d2fb2dc6e731081e1b7203953b63fe5ad373.
---
 net/rfkill/Kconfig | 5 -----
 net/rfkill/core.c  | 4 ----
 2 files changed, 9 deletions(-)

diff --git a/net/rfkill/Kconfig b/net/rfkill/Kconfig
index 51f4baeede85..eaf765876458 100644
--- a/net/rfkill/Kconfig
+++ b/net/rfkill/Kconfig
@@ -10,11 +10,6 @@ menuconfig RFKILL
 	  To compile this driver as a module, choose M here: the
 	  module will be called rfkill.
 
-config RFKILL_PM
-	bool "Power off on suspend"
-	depends on RFKILL && PM
-	default y
-
 # LED trigger support
 config RFKILL_LEDS
 	bool
diff --git a/net/rfkill/core.c b/net/rfkill/core.c
index c95475de2ceb..a001f7c1f711 100644
--- a/net/rfkill/core.c
+++ b/net/rfkill/core.c
@@ -741,7 +741,6 @@ void rfkill_pause_polling(struct rfkill *rfkill)
 }
 EXPORT_SYMBOL(rfkill_pause_polling);
 
-#ifdef CONFIG_RFKILL_PM
 void rfkill_resume_polling(struct rfkill *rfkill)
 {
 	BUG_ON(!rfkill);
@@ -776,17 +775,14 @@ static int rfkill_resume(struct device *dev)
 
 	return 0;
 }
-#endif
 
 static struct class rfkill_class = {
 	.name		= "rfkill",
 	.dev_release	= rfkill_release,
 	.dev_attrs	= rfkill_dev_attrs,
 	.dev_uevent	= rfkill_dev_uevent,
-#ifdef CONFIG_RFKILL_PM
 	.suspend	= rfkill_suspend,
 	.resume		= rfkill_resume,
-#endif
 };
 
 bool rfkill_blocked(struct rfkill *rfkill)
-- 
2.35.3

